incuna-sass/helpers/_link.sass

```diff
@@ -1,18 +1,19 @@
 // Mixin to quickly add / remove default styles to links
 @mixin link($type: default)
     cursor: pointer
-    @if $type = default
-        text-decoration: underline
-        &:hover
-            text-decoration: none
     @if $type = reverse
         text-decoration: none
         &:hover
             text-decoration: underline
-    @if $type = blank
-        text-decoration: none
-        &:hover
+    @else
```

Comment:
yeah, apparently I've been doing these wrong, each `@if` needs an `@else` or it just outputs everything and is useless. This `@else` is saying if `$type != reverse`